Source/WebCore/platform/graphics/texmap/BitmapTexture.cpp

```diff
@@ -51,8 +51,34 @@ static const GLenum s_pixelDataType = GL_UNSIGNED_INT_8_8_8_8_REV;
 static const GLenum s_pixelDataType = GL_UNSIGNED_BYTE;
 #endif
 
+// On GLES3, the format we want for packed depth stencil is GL_DEPTH24_STENCIL8, but when added through
+// the extension this format is called GL_DEPTH24_STENCIL8_OES. In any case they hold the same value 0x88F0
+// so we can just use the first one.
+// These definitions may not exist if this is a GLES1/2 context without the GL_OES_packed_depth_stencil
+// extension. We need to define the one we want to use in order to build on every case.
+#ifndef GL_DEPTH24_STENCIL8
+#define GL_DEPTH24_STENCIL8 0x88F0
+#endif
+
 namespace WebCore {
 
+GLenum depthBufferFormat()
+{
+    static std::once_flag onceFlag;
```

Comment:
TextureMapper is thread agnostic. Clients have to call all TextureMapper API from a single thread.
I think TextureMapper doesn't need thread synchronization.

Reply:
Yes, I know that TextureMapper is only used from the compositor thread. But I'm using std::call_once to ensure that the initialization of the depth buffer format happens only once. I could use a boolean flag to check whether the value is initialized or not, but std::call_once seems to be commonly used for that.

Reply:
I don't think it's worth calling this only once, since both the extension and the version are cached.